js/apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -240,9 +267,37 @@ export const UserForm = ({
               label={t("emailVerified")}
               labelIcon={t("emailVerifiedHelp")}
             />
+            {user?.attributes?.["kc.email.pending"] && (
+              <Alert
+                variant={AlertVariant.warning}
+                isInline
+                isPlain
+                title={t("emailPendingVerificationAlertTitle")}
+              >
+                The user has not confirmed the new email address{" "}
```

Comment:
This needs to be localized.

Reply:
Fixed. I did not find a way to bold the email in the message property. Is it possible to do that?